src/targets/furnace.js

```diff
@@ -0,0 +1,98 @@
+import { ClickableObject } from "../clickable-object.js";
+import { CONSTANTS } from "../constants/constants.js";
+import { characterData } from "../cookie-io.js";
+import { Button } from "../ui/button.js";
+import { getItemClass } from "../utilities.js";
+
+export class Furnace extends ClickableObject {
+    name = "Furnace";
+    varName = "bar"; // Use # bars forged as quest
+    examineText = "A red hot furnace.";
+    actions = [
+        { text: "Forge", func: "clickTarget" },
+        { text: "Examine", func: "examine" },
+    ];
+
+    constructor(scene) {
+        super();
+        this.scene = scene;
+
+        const cameraWidth = scene.cameras.main.width;
+        const cameraHeight = scene.cameras.main.height;
+        const x = cameraWidth / 2 - 230;
+        const y = cameraHeight / 2 - 150;
+        const width = 230;
+        const height = 150;
+
+        // Add invisible button for furnace
+        this.sprite = new Button(scene, x, y, width, height);
+        this.sprite.on("pointerdown", (pointer) => {
+            if (pointer.rightButtonDown()) {
+                this.createRightClickMenu(pointer.x, pointer.y, this.actions);
+            } else {
+                this.clickTarget();
+            }
+        });
+    }
+
+    async clickTarget() {
+        const inv = this.scene.dashboard.inventory;
+        const chat = this.scene.scene.get(CONSTANTS.SCENES.CHAT);
+
+        // See if ore is selected
+        const selectedIndex = inv.curSelectedItemIndex;
+        if (selectedIndex < 0) {
+            chat.writeText("Select an ore in your inventory first");
```

Comment:
Add a period here to be consistent

Reply:
Done https://github.com/EthanVieira/rs-clicker/pull/60/commits/f58a102e21838136430a3019f6ce252ac0ec0b07